cfme/cloud/provider/__init__.py

```diff
@@ -289,7 +290,8 @@ class EditTags(CFMENavigateStep):
     prerequisite = NavigateToSibling('All')
 
     def step(self, *args, **kwargs):
-        self.prerequisite_view.entities.get_entity(name=self.obj.name, surf_pages=True).check()
+        self.prerequisite_view.entities.get_entity(name=self.obj.name,
+                                                   surf_pages=True).ensure_checked()
```

Comment:
I haven't found a usage of this. It is the normal implementation of ensure_checked() that has been tested elsewhere.